mix.exs

```diff
@@ -1,17 +1,17 @@
 defmodule Torch.MixProject do
   use Mix.Project
 
+  @source_url "https://github.com/mojotech/torch"
+  @version "3.6.3"
+
   def project do
     [
       app: :torch,
-      version: "3.6.3",
+      version: @version,
       elixir: "~> 1.8",
       start_permanent: Mix.env() == :prod,
       compilers: [:phoenix, :gettext] ++ Mix.compilers(),
       name: "Torch",
-      description: "Rapid admin generator for Phoenix",
-      source_url: "https://github.com/mojotech/torch",
-      homepage_url: "https://github.com/mojotech/torch",
```

Comment:
Why are we removing this line for `homepage_url`?

Reply:
My bad. Added it back.